sdk/communication/Azure.Communication.Common/tests/Pipeline/EntraTokenGuardPolicyTests.cs

```diff
@@ -81,6 +85,60 @@ public async Task Process_WithDifferentToken_CallsProcessNext(bool async)
             }
         }
 
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task Process_WithInvalidAcsToken_CallsProcessNext(bool async)
+        {
+            // Arrange
+            var policy = new EntraTokenGuardPolicy();
+            var pipeline = new Mock<HttpPipelinePolicy>();
+            var pipelines = new ReadOnlyMemory<HttpPipelinePolicy>(new HttpPipelinePolicy[] { _httpPipelinePolicyMock.Object });
```

Comment:
Why do we need to use `ReadOnlyMemory` here?

Reply:
[HttpPipelinePolicy.ProcessNext](https://github.com/Azure/azure-sdk-for-net/blob/130d1c76e36b634d8087e0746a3e68c2f0287c43/sdk/core/Azure.Core/src/Pipeline/HttpPipelinePolicy.cs#L35) is expecting it as a param, and we can not anyhow mock it as it is a struct